homeassistant/components/wiffi/config_flow.py

```diff
@@ -38,6 +38,9 @@ async def async_step_user(self, user_input=None):
             return self._async_show_form()
 
         # received input from form or configuration.yaml
+        unique_id = f"{DOMAIN} {user_input[CONF_PORT]}"
+        await self.async_set_unique_id(unique_id)
```

Comment:
We don't have to namespace this with the domain

```suggestion
        await self.async_set_unique_id(user_input[CONF_PORT])
```